packages/components/src/Page/Page.css

```
@@ -15,3 +15,12 @@
 .narrow {
   max-width: calc(var(--base-unit) * 64);
 }
+
+.titleBar {
+  display: flex;
+  justify-content: space-between;
+}
+
+.actionGroup > :not(:last-child) {
+  margin: 0 var(--space-small) 0 0;
```

Might be easier to read if you specify…

```suggestion
.actionGroup > * {
  margin-left: var(--space-small);
```

Also, I _think_ this will allow you to avoid the `:last-child` too…

---

True, but then I am not setting the margin on the other sides... which seems less robust.

---

Discussed in person, going to use the verbose properties.